fesvr/htif.h

```diff
@@ -121,6 +124,7 @@ TARGET (RISC-V BINARY) OPTIONS\n\
 {"signature", required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 2 },     \
 {"chroot",    required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 3 },     \
 {"payload",   required_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 4 },     \
+{"size",    optional_argument, 0, HTIF_LONG_OPTIONS_OPTIND + 5 },     \
```

Comment:
I think the name "size" is too generic for this purpose.  Maybe "signature-alignment" or something?